assets/scss/custom/_navs.scss

```diff
@@ -88,4 +88,16 @@
 
 .view-switcher{
     margin-right: 15px;
-}
\ No newline at end of file
+}
+
+.checkout-wizard-active-step {
+  > a {
+    color: $brand-primary !important;
+    border-radius: $panel-border-radius !important;
+    background-color: $body-bg !important;
+    border: 1px solid $brand-primary !important;
+    &:hover {
+      background-color: $body-bg !important;
+    }
+  }
```

Comment:
May we instead replace it with `.nav-outline`, which will set `color`, `border` & `background` for `li.active > a` and introduce new variable `$nav-outline-border-radius`? Then this extension will come to bootstrap 4 unchanged.

Reply:
done